Makefile

```diff
@@ -61,7 +61,7 @@ uninstall:
 	$(call UNINSTALL_LAB_EXTENSION,@jupyterlab/toc)
 	pip uninstall -y jupyterlab-git
 	pip uninstall -y elyra
-	jupyter lab clean
+	jupyter lab clean 2>/dev/null || :
```

Comment:
I'd prefer we use the `make` syntax for ignoring errors via a dash prefix:
```
    - jupyter lab clean
```
This convention is already present near the end of file when unlinking/uninstalling the lab extension.

Reply:
Sorry, I didn't know there was such an option. Changed it